### PR TITLE
Cats 2.2.0

### PR DESCRIPTION
**Pull request by cquiroz**
scala-steward made some fixes for cats 2.2.0 but missed others. This PR completes the upgrade

### CODE REVIEW

**Review by tpolecat — APPROVED** [review]

File: modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/DividedProgress.scala [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 package seqexec.web.client.components
 
-import scala.scalajs.js.JSConverters._
 import cats.implicits._
+import scala.scalajs.js.JSConverters._
```
Comment (rpiaggio) [inline modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/DividedProgress.scala:7]: There seems to be an extra newline.

**Review by rpiaggio — APPROVED** [review]
LGTM

File: modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/DividedProgress.scala (hunk 1 above)
Comment (cquiroz) [reply modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/DividedProgress.scala:7]: Oh that's odd

**Comment by cquiroz** [comment]
I'll run some experiments with packtracker along the next few PRs.

**Pull request merged**